certbot/renewal.py

```diff
@@ -408,6 +409,16 @@ def handle_renewal_request(config):
         try:
             if renewal_candidate is None:
                 parse_failures.append(renewal_file)
+            elif lineage_config.certname  and cli.set_by_cli('autorenew'):
+                # Update renewal config file and exit.
+                symlinks = dict((kind, renewal_candidate.configuration[kind])
```

Comment:
shouldn't have to manually be setting this in the renewal config file, once it's added to the list of bools above

Reply:
I'm slightly confused here. I thought the `BOOL_CONFIG_ITEMS` was used to set the data type of the value of a field in renewalparams when certbot reads the field from the renewal config file.

I added this
```python
symlinks = dict((kind, renewal_candidate.configuration[kind])
                     for kind in storage.ALL_FOUR)
     storage.update_configuration(lineagename,
             renewal_candidate.archive_dir, symlinks, lineage_config)
```
to set `autorenew` under `[renewalparams]` in the renewal config file like:

```conf
# renew_before_expiry = 30 days
version = 0.25.0.dev0
archive_dir = /tmp/leitKHTl/conf/archive/ara.ricketyspace.net
cert = /tmp/leitKHTl/conf/live/ara.ricketyspace.net/cert.pem
privkey = /tmp/leitKHTl/conf/live/ara.ricketyspace.net/privkey.pem
chain = /tmp/leitKHTl/conf/live/ara.ricketyspace.net/chain.pem
fullchain = /tmp/leitKHTl/conf/live/ara.ricketyspace.net/fullchain.pem

# Options used in the renewal process
[renewalparams]
account = 73398b4278fc1b489d3ab10dc8b35bb5
work_dir = /tmp/leitKHTl/work
server = https://acme-staging-v02.api.letsencrypt.org/directory
authenticator = webroot
logs_dir = /tmp/leitKHTl/logs
config_dir = /tmp/leitKHTl/conf
autorenew = True
[[webroot_map]]
ara.ricketyspace.net = /home/s/pub
```

Will certbot automatically set the `autorenew` field in the renewal config file when `--autorenew` / `--no-autorenew` option is given in the commandline?